test/mpi/Makefile_cxx.mtest

```diff
@@ -12,19 +12,17 @@
 ## see Makefile.mtest for a description why this file exists, but for C++
 
 AM_CPPFLAGS = -I$(top_builddir)/include -I$(top_srcdir)/include
-LDADD = $(top_builddir)/cxx/util/mtest.o
+LDADD = $(top_builddir)/util/mtest_cxx.$(OBJEXT) $(top_builddir)/util/mtest_common.$(OBJEXT)
```

Comment:
Changing mtest_cxx.o to mtest_cxx.$(OBJEXT) is a bug-fix and should be separated into a different commit.